### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kamikaze49**
Made edits to readme file adding Getting started instructions

### CODE REVIEW

**Comment by cornzyblack** [comment]
@kangbreder there is no master branch.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
 
-[![Build Status](https://travis-ci.org/hngi/fury-backend-auth.svg?branch=develop)](https://travis-ci.org/hngi/fury-backend-auth) [![Coverage Status](https://coveralls.io/repos/github/hngi/fury-backend-auth/badge.svg?branch=develop)](https://coveralls.io/github/hngi/fury-backend-auth?branch=develop)
\ No newline at end of file
+## Getting Started
+
+### To Contribute:
+* Clone the code to your Local
+* Create a branch with the feature name you wish to work on
+* Add any changes
+* Make a pull request to master
```
Comment (kangbreder) [inline README.md:10]: Pull requests are made to develop branch and not master branch

Comment (Kamikaze49) [reply README.md]: The change has been made

Comment (Kamikaze49) [reply README.md]: The change has been made

**Pull request merged**